tests/Unit/IO/H5/Test_VolumeData.py

```diff
@@ -276,13 +284,17 @@ def test_offset_and_length_for_grid(self):
         all_extents = self.vol_file.get_extents(observation_id=obs_id)
         self.assertEqual(
             spectre_h5.offset_and_length_for_grid(
-                grid_name="grid_1",
+                grid_name="[B0(L1I0,L1I0,L1I0)]",
                 all_grid_names=all_grid_names,
                 all_extents=all_extents,
             ),
             (0, 8),
         )
 
+    def test_extend_connectivity_data_3D(self):
+        obs_ids = self.vol_file.list_observation_ids()
+        self.vol_file.extend_connectivity_data_3d(obs_ids)
```

Comment:
Can you add a small check that this did something? Like check the size of the connectivity dataset or something like that.